docs/guides/drivers/docker-container.md

```diff
@@ -45,29 +48,57 @@ WARNING: No output specified with docker-container driver. Build result will onl
 ...
 ```
 
-Note the warning "Build result will only remain in the build cache" - unlike
-the `docker` driver, the built image must be explicitly loaded into the local
-image store. We can use the `--load` flag for this:
+## Loading to local image store
+
+Unlike when using the default `docker` driver, images built with the
+`docker-container` driver must be explicitly loaded into the local image store.
+Use the `--load` flag:
 
-```console
-$ docker buildx build --load -t <image> --builder=container .
+```bash
+$ docker buildx build . --load -t <image> --builder=container
```

Comment:
Why?

Reply:
oops yeah I was not getting syntax highlighting in my editor with `console` so I used bash while editing, I will change it back...

Reply:
Or maybe you meant why I changed the position of the build context arg :-) 

only to align with the cache docs where many examples are multiline and the dot at the end becomes awkward and sometimes missed

Reply:
Having context set there is not common and does not reflect the usage:

```
Usage:  docker buildx build [OPTIONS] PATH | URL | -
```

This is what we use atm across all our documentation and examples. Would prefer to keep it as it is.